integrations/ollama/src/haystack_integrations/components/embedders/ollama/document_embedder.py

```diff
@@ -44,15 +63,12 @@ def __init__(
     def _create_json_payload(self, text: str, generation_kwargs: Optional[Dict[str, Any]]) -> Dict[str, Any]:
         """
         Returns A dictionary of JSON arguments for a POST request to an Ollama service
-          :param text: Text that is to be converted to an embedding
-          :param generation_kwargs:
```

Comment:
I would rather not document the parameter of private functions instead of duplicating/ignoring parameters that were already documented elsewhere, LMK

Reply:
Agreed.